README.markdown

```diff
@@ -7793,10 +7793,70 @@ tcpsock:setoption
 
 **context:** *rewrite_by_lua&#42;, access_by_lua&#42;, content_by_lua&#42;, ngx.timer.&#42;, ssl_certificate_by_lua&#42;, ssl_session_fetch_by_lua&#42;*
 
-This function is added for [LuaSocket](http://w3.impa.br/~diego/software/luasocket/tcp.html) API compatibility and does nothing for now. Its functionality will be implemented in future.
+This function is added for [LuaSocket](http://w3.impa.br/~diego/software/luasocket/tcp.html) API compatibility and does nothing for now. Its functionality is implemented `v0.10.18`.
 
 This feature was first introduced in the `v0.5.0rc1` release.
 
+The `option` is a string with the option name, and the value depends on the option being set:
+
+* `keepalive`
+
+	Setting this option to true enables sending of keep-alive messages on
+	connection-oriented sockets. Make sure the `connect` function
+	had been called before, for example,
+
+```lua
```

Comment:
I suggest you use inlined code snippets instead of this block snippet. Otherwise it will stop the current bulleted list as well (you need an extra level of indentation for it, but the wiki2markdown tool may not support it).

Reply:
accept